keyboards/hidtech/bastyl/keymaps/german_gaming/readme.md

```diff
@@ -0,0 +1,24 @@
+![German Gaming Layout Image](https://i.imgur.com/0y938rG.png)
```

Comment:
```suggestion
![German Gaming Layout Image](https://i.imgur.com/0y938rGh.png)
```

Reply:
All the other changes make sense to me, but this seems unnecessary and counterproductive to me.

The file size is only 283 KiB and screen resolutions are constantly increasing. Halving the size makes this image and the text fuzzy and harder to read.

Reply:
@fauxpark Thanks again for the review! :+1: 

Is this a blocking change request?

Reply:
> The file size is only 283 KiB and screen resolutions are constantly increasing. Halving the size makes this image and the text fuzzy and harder to read.
> 
> Am I missing something, and if not, is this change necessary?

I don't know if fauxpark considers this a blocking issue, but GitHub's not going to display this image at anything larger than about 1,000 pixels wide anywhere that a user would see it, no matter how large the source image is.

Reply:
@noroadsleft That does not seem to be true. Check out the way it is rendered here on a HiDPI screen:

https://github.com/qmk/qmk_firmware/blob/ecbef0f66440674cca7dac8ed6e3e940be12b6bb/keyboards/hidtech/bastyl/keymaps/german_gaming/readme.md

or

https://github.com/CreamyCookie/qmk_firmware/blob/german_gaming_keymap_for_bastyl/keyboards/hidtech/bastyl/keymaps/german_gaming/readme.md

Screenshot created using Firefox's screenshot tool (on a 1440p screen):

![hidpi](https://user-images.githubusercontent.com/3063858/107235575-18f55e80-6a25-11eb-9dd7-0eefd629a4df.png)

Now here's the fuzzy half size image again:

![0y938rGh](https://user-images.githubusercontent.com/3063858/107235607-26124d80-6a25-11eb-8079-7bb7e5e9186b.jpeg)

Even if it was true that GitHub currently downsized all images or limited the size via the layout, there's no reason to believe that this will still be the case in a couple years, especially when screen resolutions like 4K are getting really common. But, of course, `2339 x 1654` is not even that large.

I want to be clear. It's not a big issue. I'm happy for this to be merged either way, but I also believe that the change is actually counter productive for any potential user and thus also for the QMK community. (Of course, nobody might ever use the keymap, but who knows.)

Reply:
Hm, I guess this is hardware dependent; I wasn't aware GitHub did anything differently with page layout/rendering in that regard.

I personally don't find the smaller image to be so much more fuzzy as to be unreadable, but I'm undoubtedly not the best person to ask as I have 20/20 vision.

Also good point about the fact that GitHub may change this rendering in future as higher-resolution display become more common (I'm on a 24" 1080p screen).